src/inspector_profiler.h

```diff
@@ -106,7 +106,7 @@ class V8CpuProfilerConnection : public V8ProfilerConnection {
  private:
   std::unique_ptr<inspector::InspectorSession> session_;
   bool ending_ = false;
-  std::string type_ = "CPU";
+  const std::string type_ = "CPU";
```

Comment:
I think you can also just remove the members and return the C strings directly from the methods where they are used

Reply:
Good point. Fixed :)